docs/pages/documentation/3-querying-with-graql/insert-queries.md

```diff
@@ -9,9 +9,13 @@ permalink: /documentation/graql/insert-queries.html
 folder: documentation
 ---
 
-The page documents use of the Graql `insert` query, which will insert a specified [variable pattern](#variable-patterns) into the knowledge base. To follow along, or experiment further, with the examples given below, please load the *basic-genealogy.gql* file, which can be found in the *examples* directory of the Grakn installation zip, or on [Github](https://github.com/graknlabs/grakn/blob/master/grakn-dist/src/examples/basic-genealogy.gql).
+The page documents use of the Graql `insert` query, which will insert a specified [variable pattern](#variable-patterns)
+describing data into the knowledge base. To follow along, or experiment further, with the examples given below, please
```

Comment:
`The page documents use of the Graql `insert` query, which will insert a specified [variable pattern](#variable-patterns).`

Same reasons as above again